controllers/motorcontroller.py

```diff
@@ -1,36 +1,16 @@
-from TestData import*
-from canBus import*
-from Controller import*
-import sys
-class MotorController(Controller):
+import sys object
```

Comment:
Line 1: Separate imports on one line must be delimited by commas. Not sure what object is, but it is not a standard library or language provision.